### PR TITLE
Rename enum Action cases to be consistent

### PR DESCRIPTION
**Pull request by Jager-yoo**
Hi, I found that some enum `Action` cases have different names compared to most others.
It would be nice to rename those for naming consistency.

```diff
// "dismissed" should be placed at the end of the case name.

- case dismissAlert
+ case alertDismissed

- case dismissAuthorizationStateAlert
+ case authorizationStateAlertDismissed


// also rename enum for cancellable id, like the others.

- enum CancelID {}
+ enum CancelId {}
```

Tests of the changed files have all passed.
Please let me know what you think. Thanks! 😃

### CODE REVIEW

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift [hunk 1]
```diff
@@ -61,14 +61,14 @@ struct AnimationsEnvironment {
 
 let animationsReducer = Reducer<AnimationsState, AnimationsAction, AnimationsEnvironment> {
   state, action, environment in
-  enum CancelID {}
+  enum CancelId {}
```
Comment (stephencelis) [inline Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:64]: We've recently changed these from `Id` to `ID` on the `concurrency-beta` branch, which will be merged soon. Care to revert these changes?
```suggestion
  enum CancelID {}
```

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift [hunk 2]
```diff
@@ -47,7 +47,7 @@ struct AnimationsState: Equatable {
 
 enum AnimationsAction: Equatable {
   case circleScaleToggleChanged(Bool)
-  case dismissAlert
+  case alertDismissed
```
Comment (stephencelis) [inline Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:50]: Can you please alphabetize this so it comes first now?

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift [hunk 3]
```diff
@@ -61,14 +61,14 @@ struct AnimationsEnvironment {
...
-  enum CancelID {}
+  enum CancelId {}
 
   switch action {
   case let .circleScaleToggleChanged(isScaled):
     state.isCircleScaled = isScaled
     return .none
 
-  case .dismissAlert:
+  case .alertDismissed:
```
Comment (stephencelis) [inline Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:71]: Can you also re-alphabetize this case so it's handled first?

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift [hunk 4]
```diff
@@ -78,7 +78,7 @@ let animationsReducer = Reducer<AnimationsState, AnimationsAction, AnimationsEnv
         .map { (output: .setColor($0), duration: 1) },
       scheduler: environment.mainQueue.animation(.linear)
     )
-    .cancellable(id: CancelID.self)
+    .cancellable(id: CancelId.self)
```
Comment (stephencelis) [inline Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:81]:
```suggestion
    .cancellable(id: CancelID.self)
```

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift [hunk 5]
```diff
@@ -93,7 +93,7 @@ let animationsReducer = Reducer<AnimationsState, AnimationsAction, AnimationsEnv
 
   case .resetConfirmationButtonTapped:
     state = AnimationsState()
-    return .cancel(id: CancelID.self)
+    return .cancel(id: CancelId.self)
```
Comment (stephencelis) [inline Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:96]:
```suggestion
    return .cancel(id: CancelID.self)
```

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift (hunk 1 above)
Comment (Jager-yoo) [reply Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift]: oh, I didn't catch that branch, sorry.
I confirmed you already reverted `CancelId` → `CancelID`. Thanks!

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift (hunk 2 above)
Comment (Jager-yoo) [reply Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:50]: Should I change it "dismissAlert" back? but in `concurrency-updates-proto` branch, "dismissAlert" case is found in only 01-GettingStarted-Animations.swift. Can you tell me why?

Comment (stephencelis) [reply Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift:50]: I wouldn't worry about any experimental branches past `concurrency-updates` for now :) Some of those future branches are able to simplify cases away, which may explain that omission, but I need to refresh my memory to understand why that particular case is missing.

Comment (Jager-yoo) [reply Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift]: Got it. I reverted this case in https://github.com/pointfreeco/swift-composable-architecture/pull/1240/commits/be713f8f2d284d4c4ee9c6daff9c17490b6b29b1 commit. Please review again.
I'm expecting new concurrency features! Thank you for maintaining this well designed library. 👍🏻

File: Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift [hunk 6]
```diff
@@ -47,7 +47,7 @@ struct AnimationsState: Equatable {
 
 enum AnimationsAction: Equatable {
   case circleScaleToggleChanged(Bool)
-  case alertDismissed
```
Comment (tgrapperon) [inline Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift]: I think that what @stephencelis what suggesting was to reorder cases in the alphabetical order:
```swift
case alertDismissed
case circleScaleToggleChanged
…
```

Comment (Jager-yoo) [reply Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift]: That's what he meant. Thanks @tgrapperon ! 😄 just edited last commit.

**Review by stephencelis — APPROVED** [review]

**Comment by stephencelis** [comment]
Thanks!
